libc/src/__support/threads/linux/mutex.h

```diff
@@ -116,7 +116,26 @@ struct Mutex {
     }
   }
 
-  MutexError trylock();
+  MutexError trylock() {
+    FutexWordType mutex_status = FutexWordType(LockState::Free);
+    FutexWordType locked_status = FutexWordType(LockState::Locked);
+
+    if (futex_word.compare_exchange_strong(mutex_status,
+                                           FutexWordType(LockState::Locked))) {
+      return MutexError::NONE;
+    }
```

Comment:
```suggestion
    if (futex_word.compare_exchange_strong(mutex_status,
                                           FutexWordType(LockState::Locked)))
      return MutexError::NONE;
```
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements